editor/src/components/canvas/canvas-utils.ts

```diff
@@ -2964,7 +2967,7 @@ export function getValidElementPathsFromElement(
       ),
     )
 
-    const name = getJSXElementNameAsString(element.name)
+    const name = isJSXElement(element) ? getJSXElementNameAsString(element.name) : 'Fragment'
```

Comment:
This logic feels like it should probably be wrapped up in a function, but that can be done subsequently.

Reply:
It might be worth doing that now, because there's quite a significant implicit assumption being made here, since right now this is applying to an element that `isJSXElementLikeWithChildren`, which could easily be made explicit by converting this to a function.

Reply:
Added to backlog.